templates/pageLevelProgressItem.jsx

```diff
@@ -64,6 +64,12 @@ export default function PageLevelProgressItem(props) {
           </span>
         </span>
 
+        {_isLocked &&
```

Comment:
@kirsty-hames Would it make sense to _always_ include this icon? That way, if we do want to add a checkmark to indicate completion, this would only require theme .less work instead of a template override.

Reply:
Amended in [e494553](https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/pull/210/commits/e4945538cc07c609dc20220a1bdd949220da2c94).